src/util/Color.js

```diff
@@ -338,13 +339,13 @@ define([
         };
 
         /**
-         * Create a rgba color string that CSS can use.
+         * Create a rgba color string that conforms to CSS Module Level 3 specification.
          * @returns {string} A color string suitable for CSS.
          */
-        Color.prototype.toRGBAString = function () {
-            var red = Math.floor(this.red * 255),
-                green = Math.floor(this.green * 255),
-                blue = Math.floor(this.blue * 255);
+        Color.prototype.toCSSString = function () {
+            var red = Math.round(this.red * 255),
+                green = Math.round(this.green * 255),
+                blue = Math.round(this.blue * 255);
 
             return 'rgba(' + red + ' ,' + green + ' ,' + blue + ' ,' + this.alpha + ')';
```

Comment:
@zglueck Can we include a brief comment pointing out that the alpha channel is correctly inconsistent with the other channels?

Reply:
no problem!